article/Experiment/GCF-ablation-study.tex

```diff
@@ -0,0 +1,99 @@
+When LightGCN is tested with concatenation as layer combination it is called \textit{LightGCN-concat}
+\begin{equation}
+    \mathbf{e}_{u}^{(k+1)} = \sum^{}_{i \in \mathcal{N}_u}  \frac{1}{\sqrt{|\mathcal{N}_u||\mathcal{N}_i|}}\left( \mathbf{e}_i^{(k)} + \mathbf{e}_i^{(k)} \odot \mathbf{e}_u^{(k)} \right)
+    \label{eq:GCF-minus-sc}
+\end{equation}
+\begin{equation}
+    \mathbf{e}_{u}^{(k+1)} = \mathbf{e}_{u}^{(k)} + \sum^{}_{i \in \mathcal{N}_u}  \frac{1}{\sqrt{|\mathcal{N}_u||\mathcal{N}_i|}} \mathbf{e}_i^{(k)} \odot \mathbf{e}_u^{(k)}
+    \label{eq:GCF-only-IP}
+\end{equation}
+\begin{equation}
+    \mathbf{e}_{u}^{(k+1)} = \mathbf{e}_{u}^{(k)} + \sum^{}_{i \in \mathcal{N}_u}  \frac{1}{\sqrt{|\mathcal{N}_u||\mathcal{N}_i|}} \mathbf{e}_i^{(k)}
+    \label{eq:GCF-minus-IP}
+\end{equation}
+\autoref{fig:GCF-NDCG-ablation-study} and \autoref{fig:GCF-recall-ablation-study} contains the GCF ablation study, where all GCF modifications still have the inner product between users and items, except for LightGCN.
+The methods are as described here:\\
+\begin{itemize}
+    \item \textbf{GCF}: The original GCF method as described in \autoref{subsubsec:GCF-embed-propagation}.
+    \item \textbf{GCF-minus-sc}: GCF without self connections
+    \item \textbf{GCF-only-IP}:  GCF where $e_i^{(k)}$ has been removed in \autoref{eq:GCF-embedding}, so that GCF's graph convolutions only considers the inner product of users and items.
+    \item \textbf{GCF-only-IP-minus-sc}: Implemented as GCF-only-ip but without self connections.
+    \item \textbf{GCF-minus-IP}: GCF where inner product has been removed.
+    \item \textbf{GCF-minus-IP-and-sc}: Same as GCF-minus-IP, but without self connections.
+    \item \textbf{LightGCN-concat}: LightGCN with concatenation as layer combination.
+    \item \textbf{LightGCN}: Original LightGCN as described in \autoref{subsubsec:LightGCN-embed-propagation}.
+    \item \textbf{LightGCN-plus-sc}: LightGCN, but with self connections.
+    \item \textbf{GCF-sum-only-IP}: Implemented as GCF-only-IP except that the layer combination method used is weighted summation.
+    \item \textbf{GCF-sum}: GCF where the layer combination has been changed to weighted summation instead of concatenation.
+    \item \textbf{GCF-sum-minus-sc}: Implemented as GCF-sum but without self connections.
+\end{itemize}
+\begin{figure}[h!]
+    \includegraphics[width=\linewidth]{figures/gcf-all-ndcg.png}
+    \caption{NDCG@50 for the Yelp2020 dataset.}
+    \label{fig:GCF-NDCG-ablation-study}
+\end{figure}
+\begin{figure}[h!]
+    \includegraphics[width=\linewidth]{figures/gcf-all-recall.png}
+    \caption{Recall@50 on the Yelp2020 dataset.}
+    \label{fig:GCF-recall-ablation-study}
+\end{figure}
+From \autoref{fig:GCF-NDCG-ablation-study} and \autoref{fig:GCF-recall-ablation-study} it can be seen that the methods that do utilize concatenation as layer combination generally performs worse than the methods that utilize summation.
```

Comment:
```suggestion
From \autoref{fig:GCF-NDCG-ablation-study} and \autoref{fig:GCF-recall-ablation-study} it can be seen that the methods that do utilize concatenation as layer combination generally performs worse than the methods that utilize weigthed summation.
```